### PR TITLE
create the displayMessage function

### PR DESCRIPTION
**Pull request by mattlub**
includes arduino setup using johnny-five module.

### CODE REVIEW

File: src/arduino/display_message.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var five = require("johnny-five")
```
Comment (Karyum) [inline src/arduino/display_message.js]: const here instead of var

File: src/arduino/display_message.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+var board = new five.Board({
```
Comment (Karyum) [inline src/arduino/display_message.js]: also here

File: src/arduino/display_message.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  repl: false
+})
+
+board.on("ready", function() {
```
Comment (Karyum) [inline src/arduino/display_message.js]: arrow function here

File: src/arduino/display_message.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  console.log('board ready')
```
Comment (Karyum) [inline src/arduino/display_message.js]: remove this log

File: src/arduino/display_message.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  var display = new five.Led.Matrix({
```
Comment (Karyum) [inline src/arduino/display_message.js]: const

File: src/arduino/display_message.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    pins: {
+      data: 2,
+      clock: 3,
+      cs: 4
+    }
+  })
+  display.on()
+})
+
+function displayMessage(display, message) {
```
Comment (Karyum) [inline src/arduino/display_message.js]: arrow function

File: src/arduino/display_message.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  displayLetters(display, message.split(""))
+}
+
+function displayLetters(display, letters) {
```
Comment (Karyum) [inline src/arduino/display_message.js]: arrow function

**Review by Karyum — CHANGES_REQUESTED** [review]
convert to es6

**Review by Karyum — APPROVED** [review]
honestly i barely understand whats going on here but looks pretty pro :sweat_smile: :+1:

**Review by mantagen — APPROVED** [review]

**Pull request merged**